feature/bgp/addpath/ate_tests/route_propagation_test/route_propagation_test.go

```diff
@@ -154,9 +154,43 @@ func TestBGP(t *testing.T) {
 		desc:     "propagate IPv4 over IPv4",
 		fullDesc: "Advertise prefixes from ATE port1, observe received prefixes at ATE port2",
 		dut: dutData{&telemetry.NetworkInstance_Protocol_Bgp{
+			PeerGroup: map[string]*telemetry.NetworkInstance_Protocol_Bgp_PeerGroup{
+				"BGP-PEER-GROUP1": {
+					PeerGroupName: ygot.String("BGP-PEER-GROUP1"),
+					AfiSafi: map[telemetry.E_BgpTypes_AFI_SAFI_TYPE]*telemetry.NetworkInstance_Protocol_Bgp_PeerGroup_AfiSafi{
+						telemetry.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST: {
+							AfiSafiName: telemetry.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST,
+							Enabled:     ygot.Bool(true),
+						},
+					},
+					ApplyPolicy: &telemetry.NetworkInstance_Protocol_Bgp_PeerGroup_ApplyPolicy{
+						ExportPolicy: []string{"PERMIT-ALL"},
```

Comment:
We need to configure PERMIT-ALL policy as well.

Reply:
@Hobbydos please check.

Reply:
those were added some time back in latest commit - See: https://github.com/openconfig/featureprofiles/blob/a39ddda34a80c6f697fb91e918f32a3c452ef78f/feature/bgp/addpath/ate_tests/route_propagation_test/route_propagation_test.go#L121-L126